projects/tad/index.md

```diff
@@ -0,0 +1,84 @@
+|      | Reliability            | The tool handles errors gracefully and informs users of any issues. | S        |
+|      | Reliability            | The tool provides clear error messages and instructions for troubleshooting. | S        |
+|      | Usability              | The tool possess a clean, intuitive, and visually appealing UI that follows industry standards. | S        |
+|      | Usability              | The tool offers keyboard shortcuts for efficient interaction. | C        |
+|      | Usability              | The tool provides clear and consistent navigation, making it easy for users to find what they need. | S        |
+|      | Usability              | The tool is responsive and provides instant feedback.        | S        |
+|      | Usability              | The user interface is multilingual and supports at least English. | S        |
+|      | Usability              | The user interface can easily be translated into other languages. | C        |
+|      | Help & Documentation   | The tool provides comprehensive online help documentation with searchable functionalities. | S        |
+|      | Help & Documentation   | The tool offers context-sensitive help within the application. | C        |
+|      | Help & Documentation   | The online documentation includes video tutorials and training materials for ease of learning. | C        |
+|      | Help & Documentation   | The project provides readily available customer support through various channels  (e.g., email, phone, online chat) to address user inquiries and troubleshoot issues. | C        |
+|      | Performance Efficiency | The tool operates efficiently and minimize resource utilization. | M        |
+|      | Performance Efficiency | The tool responds to user actions instantly.                 | M        |
+|      | Performance Efficiency | Resource consumption of the tool itself (e.g., CPU, memory) does not exceed [Threshold] under normal load. Resource consumption of the tests can not be predicted. | C        |
+|      | Performance Efficiency | The tool is scalable to accommodate increased user base and data volume. | S        |
+|      | Maintainability        | The project is open source.                                  | S        |
+|      | Maintainability        | It is possible to contribute to the source.                  | S        |
+|      | Maintainability        | The tool is easy to modify and maintain.                     | M        |
+|      | Maintainability        | The tool adheres to industry coding standards and best practices to ensure code quality and maintainability. | M        |
+|      | Maintainability        | The system is modular, allowing for easy modification of individual components. | S        |
+|      | Maintainability        | Diagnostic tools are available to identify and troubleshoot issues. | S        |
+|      | Maintainability        | The code is written in a modern and common programming language. | M        |
```

Comment:
Maybe drop the common? I like the modern as we wouldn't like tools in Fortran, but more specific languages could be Go which would also not be a game breaker it think (or would you still call this a common language?

Reply:
I think R is common enough... both common en modern are open to discussion ofcourse.. we could make it more specific (but that may make it too narrow):

The code is written in a common, widely adopted and supported and actively used and maintained programming language.